parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -4582,3 +4584,65 @@ AlterProcedureWithOption AlterProcedureWithOption() :
         <NO> <WARNING> { return AlterProcedureWithOption.NO_WARNING; }
     )
 }
+
+SqlNode ConditionalStmt() :
+{
+    final SqlNode e;
+}
+{
+    e = IfStmt()
+    { return e; }
+}
+
+SqlIfStmt IfStmt() :
+{
+    SqlNode e;
+    final SqlNodeList conditionMultiStmtList = new SqlNodeList(getPos());
+    final SqlStatementList elseMultiStmtList = new SqlStatementList(getPos());
+}
+{
+    <IF> e = ConditionMultiStmtPair()
+    { conditionMultiStmtList.add(e); }
+    (
+        <ELSE> <IF> e = ConditionMultiStmtPair()
+        { conditionMultiStmtList.add(e); }
+    )*
+    [
+        <ELSE>
+        CreateProcedureStmtList(elseMultiStmtList)
+    ]
+    <END> <IF>
+    {
+        return new SqlIfStmt(getPos(), conditionMultiStmtList,
+            elseMultiStmtList);
+    }
+}
+
+SqlNode ConditionMultiStmtPair() :
+{
+    final SqlNode condition;
+    final SqlStatementList multiStmtList = new SqlStatementList(getPos());
+    SqlNode e;
```

Comment:
nit: Remove unreferenced variable.

Reply:
Fixed in https://github.com/googleinterns/calcite/pull/245/commits/01b6e805a3c5c8a60af83236b5e706211afdff7c